docs/content/tutorials/building-comfort/_index.md

```diff
@@ -0,0 +1,1161 @@
+```
+
+Note that reactions might take some time to come up. You can use the following
+  convenient command to wait for it (upto 120 seconds) to come up:
+
+```sh
+drasi wait reaction building-signalr-hub -t 120
+```
+
+Once a reaction is online you should be able to see it like so:
+
+```sh
+drasi list reaction
+```
+
+Following should be the output with nothing in the 'Messages' column:
+```
+           ID          | AVAILABLE | MESSAGES  
+-----------------------+-----------+-----------
+  building-signalr-hub | true      |           
+```
+
+##### Troubleshooting the reaction
+Check messages in output of `drasi list reaction`.
+
+For more details you can also use
+  `drasi describe reaction building-signalr-hub`.
+
+Check the logs of pods created for the services `building-signalr-hub-gateway`.
+  These are deployed under drasi's namespace (default: drasi-system) in your
+  kubernetes cluster. You should also investigate the logs of the pods
+  created for source which will have names like
+  `building-facilities-reactivator-xxxxx` or
+  `building-facilities-proxy-xxxxx`.
+
+
+##### Port forwarding for Dashboard
+
+The Drasi reaction creates a Kubernetes service `building-signalr-hub-gateway` in the namespace `drasi-system`. This is the SignalR hub our frontend accessible within the Kubernetes cluster.
+
+Since the Dashboard will run in your browser, you will need port forwarding. Run the following command in the terminal.
+
+```sh
+kubectl -n drasi-system port-forward svc/building-signalr-hub-gateway 8080:8080
+```
+
+This will ensure that the SignalR hub can be accessed from localhost on port 8080. 
+
+{{< tabpane >}}
+{{% tab header="Github Codespaces" text=true %}}
+
+Github will also automatically create URL like
+  `http://<your-codespace-url>-8080.app.github.dev`. This port needs to be
+  made public by by right clicking on port 8080 on the `PORTS` tab like so:
+
+{{< figure src="Screenshot_PortVisibility.png" alt="Screenshot showing how to make port public" width="50%" >}}
+
+Once done, your `8080` port entry in `PORTS` tab should look like this:
+
+{{< figure src="Screenshot_Public8080.png" alt="Screenshot showing port 8080 being public" width="100%" >}}
+
+If all works well, you should be able to access the hub URL:
+  `https://<your-codespace-url>-8080.app.github.dev/hub` in your browser
+  and see `Connection ID required` on a blank page.
+
+{{% /tab %}}
+{{% tab header="VS Code Dev Container" text=true %}}
+
+VS Code will automatically forward port 8080 to localhost.
+  You should be able to access the hub URL:
+  `http://localhost:8080.app.github.dev/hub` in your browser and see
+  `Connection ID required` on a blank page.
+
+{{% /tab %}}
+{{% tab header="Local setup" text=true %}}
+
+No extra step should be needed if you are going to run the realtime dashboard
+  locally. You should be able to access the hub URL:
+  `http://localhost:8080.app.github.dev/hub` in your browser and see
```

Comment:
Shouldn't the URL be something like `http://localhost:8080/hub`? I am just not sure where the github part came from

Reply:
fixed.